src/derivative_utils.jl

```diff
@@ -302,9 +302,22 @@ function SciMLOperators.update_coefficients!(W::WOperator,
     dtgamma = nothing,
     transform = nothing)
     if (u !== nothing) && (p !== nothing) && (t !== nothing)
-        update_coefficients!(W.J, u, p, t)
+        if W.J isa FunctionOperator
+            update_coefficients!(W.J, u, ifelse(W.J.p === nothing, nothing, p),
+                ifelse(W.J.t === nothing, nothing, t))
```

Comment:
what's this all about? Why doesn't the `update_coefficients!` handle this correctly?